assets/src/components/editing/editor/normalizers/normalizer.ts

```diff
@@ -92,12 +90,16 @@ export function installNormalizer(
       }
 
       if (options.spacesNormalize && spacesNormalize(editor, node, path)) return;
+
+      // 8/9/2023 - When code elements were set to isVoid=true, these normalizers stopped doing anything
```

Comment:
As noted, the code normalizer didn't do anything ever since we set isVoid:true on those